scripts/cibuild/BuildValidator.ps1

```diff
@@ -51,6 +51,6 @@ if ($BuildRTM -eq 'false')
         & $NuGetValidator localization --artifacts-path $ArtifactsLocation --output-path $ArtifactsLogOutputDir --comments-path $LocalizationRepository
     }
 
-    # We want to exit the process with success even if there are errors.
-    exit 0
+    # return the exit code from the validator
```

Comment:
when does this run again?
Would it prevent the tests from running? And if yes, which ones?

Reply:
ok. 
So it's basically a change that could introduce more failures, but probably for the right reason. 

We need to watch out that it doesn't get too noisy.